training-service/src/app/pages/about/about.component.css

```diff
@@ -1,4 +1,19 @@
-.col-sm{
-    display: table;
-    margin:auto;
-}
+.image-padding {
```

Comment:
about.component.css 
please change to 
scss file

Reply:
done